cli/cmd/sync_v3.go

```diff
@@ -249,3 +242,28 @@ func syncConnectionV3(ctx context.Context, sourceClient *managedplugin.Client, d
 	fmt.Printf("%s. Resources: %d, Errors: %d, Warnings: %d, Time: %s\n", msg, totalResources, totals.Errors, totals.Warnings, syncTimeTook.Truncate(time.Second).String())
 	return nil
 }
+
+func tableNameFromSchema(sc *arrow.Schema) string {
+	tableName, _ := sc.Metadata().GetValue("cq:table_name")
+	return tableName
+}
+
+func deleteStale(client plugin.Plugin_WriteClient, tables map[string]bool, sourceName string, syncTime time.Time) error {
+	for tableName := range tables {
+		// TODO remove this print
+		fmt.Println("Deleting stale records from table", tableName)
```

Comment:
Should we remove the print now?

Reply:
yeah going to remove once proto-buf PR is merged